### PR TITLE
feat: Adding filename normalizer function

### PR DESCRIPTION
**Pull request by KillianCourvoisier**
Adding a function to normalize filename in the ContentScript for downloaded konnector's files

#### Checklist

Before merging this PR, the following things must have been done:

* [ ] Faithful integration of the mockups at all screen sizes
* [ ] Tested on iOS
* [ ] Tested on Android
* [ ] Localized in English and French
* [ ] All changes have test coverage
* [ ] Updated README & CHANGELOG, if necessary

### CODE REVIEW

**Comment by doubleface** [comment]
Ce serait  bien d'ajouter des tests unitaires pour cette fonction pour qu'on voit exactement comment elle est utilisée

**Comment by KillianCourvoisier** [comment]
@doubleface Ping pour review :pray:

File: connectors/connectorLibs/ContentScript.js [hunk 1]
```diff
@@ -401,6 +402,40 @@ export default class ContentScript {
    * @returns {Object} : Connector execution result. TBD
    */
   async fetch({ context }) {}
+
+  /**
+   * Filename normalization function, harmonize the filename format between all connectors
+   *
+   * @param {Object} filenameInfos : Object containing year/month/day and the vendor's name at least. It can contain an amount, a documentType and a vendorRef if needed
```
Comment (Crash--) [inline connectors/connectorLibs/ContentScript.js:409]: You should define the Object. 🙏 
```
* @param {object} filenameInfos               filenameInfos
 * @param {string}   filenameInfos.year    Year of... 
 *
```

If you have an optional param: 
```
 * @param {string}    [filenameInfos.amount]  Amount of... 
 ```

**Comment by KillianCourvoisier** [comment]
Ping @Crash--

**Review by acezard — APPROVED** [review]

**Pull request merged**